lib/collection/src/config.rs

```diff
@@ -48,6 +48,13 @@ pub struct CollectionParams {
     pub vector_size: usize,
     /// Type of distance function used for measuring distance between vectors
     pub distance: Distance,
+    /// Number of shards the collection has
+    #[serde(default = "default_shard_number")]
+    pub shard_number: u32,
```

Comment:
Need to generate OpenAPI docs `bash -x tools/generate_openapi_models.sh`
And also update gRPC interface and docs `bash -x tools/generate_grpc_docs.sh`